packages/debug/src/node/debug-model.ts

```diff
@@ -87,6 +87,11 @@ export interface DebugAdapterForkExecutable {
  */
 export type DebugAdapterExecutable = DebugAdapterSpawnExecutable | DebugAdapterForkExecutable;
 
+export interface DebugProtocolMessage extends Object {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [properties: string]: any;
```

Comment:
Is this `any` necessary to align with a vscode type? Alternatively I would change this to `unknown`.

Reply:
It's my attempt to convince Typescript that I want an object type, not a string. Any suggestions on how to express that better would be welcome.

Reply:
I would usually use `Record<string, unknown>` or something similar for that. `any` always accepts any type, not only objects. The record should probably be what you're looking for.